src/libraries/Microsoft.PowerFx.Core/Parser/ParseResult.cs

```diff
@@ -106,6 +100,8 @@ internal ParseResult(TexlNode root, List<TexlError> errors, bool hasError, List<
             After = after;
 
             Text = text;
+
+            ErrorMessageLocale = errorMessageLocale ?? CultureInfo.CurrentCulture;
```

Comment:
> ?? CultureInfo.CurrentCulture [](http://example.com/codeflow?start=51&length=30)

why?
Changing from null to non-null could be significant.

Reply:
code simplification

Reply:
Can you elaborate?